src/Lykke.Service.EasyBuy.DomainServices/PricesService.cs

```diff
@@ -21,6 +23,10 @@ public class PricesService : IPricesService
         private readonly IPricesRepository _pricesRepository;
         private readonly ISettingsService _settingsService;
         
+        private readonly ConcurrentDictionary<string, Price> _latestPrices;
+        private readonly ConcurrentDictionary<string, SemaphoreSlim> _semaphores;
```

Comment:
Why do we need a collection of semaphores?
Why not just one lock object for the whole _latestPrices?